airbyte-tests/src/automaticMigrationAcceptanceTest/resources/docker-compose-migration-test-first-run.yaml

```diff
@@ -0,0 +1,62 @@
+version: "3.7"
```

Comment:
maybe put a comment in the top of this file explaining where it came from. i thinks its the docker compose file from 0.14.2 or something, right?